src/Raven.Server/Documents/Handlers/CountersHandler.cs

```diff
@@ -261,6 +263,13 @@ public async Task Batch()
 
                 var counterBatch = JsonDeserializationClient.CounterBatch(countersBlittable);
 
+                if (TrafficWatchManager.HasRegisteredClients)
+                {
+                    var sb = new StringBuilder();
+                    sb.Append(/*"Counter:\n"+ */countersBlittable);
```

Comment:
What is this ? `/*"Counter:\n"+ */`

Reply:
Can just call `countersBlittable.ToString()` here, no?

Reply:
I didnt know if raw data is needed, or formatted with a title, like this:
Query:
.....
Parameters:
.....

So i kept it commented.

There are few places that I can use HttpContext.Items["TrafficWatch"] = String / .toString()
directly.